src/vs/workbench/services/extensionManagement/browser/webExtensionsScannerService.ts

```diff
@@ -676,13 +706,30 @@ export class WebExtensionsScannerService extends Disposable implements IWebExten
 						this.logService.info('Ignoring invalid extension while scanning', storedWebExtensions);
 						continue;
 					}
+					let packageNLSUris: Map<string, URI> | undefined;
+					if (e.packageNLSUris) {
+						packageNLSUris = new Map<string, URI>();
+						forEach<UriComponents>(e.packageNLSUris, (entry) => packageNLSUris!.set(entry.key, URI.revive(entry.value)));
+					}
+
+					const location = URI.revive(e.location);
+
+					// Migrate to new format. Delete after some time.
+					let fallbackPackageNLSUri = e.fallbackPackageNLSUri;
+					if ((e as any).packageNLSUri) {
```

Comment:
Also check for `packageNLSUris` and if exists and do not trigger for migration. Might not happen but this makes it look complete as the migration is to populate `packageNLSUris` property.